src/XamlStyler/StylerService.cs

```diff
@@ -19,18 +21,65 @@ namespace Xavalon.XamlStyler
 {
     public class StylerService
     {
+        private readonly string[] ignoredNamespacesInOrdering = new string[]
```

Comment:
designNamespaces?

Reply:
Well , I started implementation with general purposae in mind - to ignore chosen namespaces when ordering attributes (it was up to user which namespaces to ignore). So, now it makes sense to name it like you suggested, but I'd like to keep that name, as it perfectly explains itself.